examples/collection-only/collection.json

```diff
@@ -83,11 +82,6 @@
       "minimum": 6.78,
       "maximum": 89.9
     },
-    "sci:citation": {
```

Comment:
Bummer, that was an interesting use case where you actually define a pattern for values. We should try to come up with such a use case for another extension field.